builtin/common/serialize.lua

```diff
@@ -79,6 +79,7 @@ local function serialize(value, write)
 			if type_ == "table" then
 				write("{}")
 			elseif type_ == "function" then
+				core.log("deprecated", "Support for functions in `core.serialize` is deprecated.")
```

Comment:
isn't this duplicate?

Reply:
good catch, i didn't mean to commit this.